kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/NetworkAdminServiceImpl.java

```diff
@@ -1506,57 +1506,57 @@ private void enableWifiInterface(String ifaceName, NetInterfaceStatus status, Wi
             return;
         }
 
-        s_logger.debug("Configuring {} for {} mode", ifaceName, wifiMode);
+        logger.debug("Configuring {} for {} mode", ifaceName, wifiMode);
 
-        s_logger.debug("Stopping hostapd and wpa_supplicant");
+        logger.debug("Stopping hostapd and wpa_supplicant");
         HostapdManager.stop(ifaceName);
         WpaSupplicantManager.stop(ifaceName);
 
         if (status == NetInterfaceStatus.netIPv4StatusEnabledLAN && wifiMode.equals(WifiMode.MASTER)) {
 
-            s_logger.debug("Starting hostapd");
+            logger.debug("Starting hostapd");
             HostapdManager.start(ifaceName);
 
         } else if ((status == NetInterfaceStatus.netIPv4StatusEnabledLAN
                 || status == NetInterfaceStatus.netIPv4StatusEnabledWAN)
                 && (wifiMode.equals(WifiMode.INFRA) || wifiMode.equals(WifiMode.ADHOC))) {
 
             if (wifiConfig != null) {
-                s_logger.debug("Starting wpa_supplicant");
+                logger.debug("Starting wpa_supplicant");
                 WpaSupplicantManager.start(ifaceName, wifiMode, wifiConfig.getDriver());
                 if (isWifiConnectionCompleted(ifaceName, 60)) {
-                    s_logger.debug("WiFi Connection Completed on {} !", ifaceName);
+                    logger.debug("WiFi Connection Completed on {} !", ifaceName);
                 } else {
-                    s_logger.warn("Failed to complete WiFi Connection on {}", ifaceName);
+                    logger.warn("Failed to complete WiFi Connection on {}", ifaceName);
                 }
             } else {
-                s_logger.warn("No WifiConfig configured for mode " + wifiMode);
+                logger.warn("No WifiConfig configured for mode " + wifiMode);
             }
         } else {
-            s_logger.debug("Invalid wifi configuration - NetInterfaceStatus: {}, WifiMode:{}", status, wifiMode);
+            logger.debug("Invalid wifi configuration - NetInterfaceStatus: {}, WifiMode:{}", status, wifiMode);
         }
     }
 
     private void disableWifiInterface(String ifaceName) throws KuraException {
-        s_logger.debug("Stopping hostapd and wpa_supplicant");
+        logger.debug("Stopping hostapd and wpa_supplicant");
         HostapdManager.stop(ifaceName);
         WpaSupplicantManager.stop(ifaceName);
     }
 
     // Submit new configuration, waiting for network configuration change event before returning
     private void submitNetworkConfiguration(List<String> modifiedInterfaceNames,
             NetworkConfiguration networkConfiguration) throws KuraException {
-        short timeout = 30;		// in seconds
+        double timeout = 30;		// in seconds
```

Comment:
@cdealti @MMaiero I think the idea here was to have a timeout of 30 seconds but since timeout was declared as short and Thread.sleep was set to 500 msec we ended up having a timeout of 15 seconds since decrementing a short by 0.5 decrements it by 1. So I think it would be ok to declare timeout as float.